# Go To Sleep Command

bool enable
float32[] cmd
---
